<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1001</width>
      <height>801</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>421</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*background-color:rgb(56,41,74);*/
background-color: rgb(0, 0, 0);
border-top-left-radius:0px;
border-bottom-left-radius:0px;

border-top-right-radius:0px;
border-bottom-right-radius:0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>471</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 57, 84);
border-top-right-radius:0px;
border-bottom-right-radius:0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(255,255,255,200);</string>
     </property>
     <property name="text">
      <string>Admin Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>230</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(22,34,55,66);
border:5px solid rgba(0,0,0,0);
border-bottom-color: rgba(85, 85, 255,100);
color:rgb(255,255,255);
padding-bottom: 5px;

</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>280</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(22,34,55,66);
border:5px solid rgba(0,0,0,0);
border-bottom-color: rgba(85, 85, 255,100);
color:rgb(255,255,255);
padding-bottom: 5px;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>370</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 85, 127);
border-radius:0px;
color:white;
font-weight:bold;
}

QPushButton:hover{
background-color: rgb(100, 85, 127);
border-radius:0px;
color:white;
font-weight:bold;
}
</string>
     </property>
     <property name="text">
      <string>L O G I N</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>421</width>
       <height>291</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>NSimSun</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(255,255,255,200);</string>
     </property>
     <property name="text">
      <string>CCTV Based
 Restricted Area Monitoring System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>370</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(0, 85, 127);
border-radius:0px;
color:white;
font-weight:bold;
}

QPushButton:hover{
background-color: rgb(100, 85, 127);
border-radius:0px;
color:white;
font-weight:bold;
}
</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>110</y>
       <width>171</width>
       <height>171</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>NSimSun</family>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgba(255,255,255,200);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../graphics/resource.qrc">:/icon/icon/dome_camera_80px.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../graphics/resource.qrc"/>
 </resources>
 <connections/>
</ui>
